apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,54 @@
 import { Component, OnInit } from '@angular/core';
+import { CityStore } from '../../data-access/city.store';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { City } from '../../model/city.model';
+import { CardComponent } from '../../ui/card/card.component';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `
+    <app-card
+      [list]="cities"
+      nameKey="name"
+      customClass="bg-light-blue"
+      (addItem)="onAddItem()"
+      (deleteItem)="onDeleteItem($event)">
+      <img image-header src="assets/img/city.png" width="200px" />
+    </app-card>
+  `,
+  styles: [
+    `
+      ::ng-deep .bg-light-blue {
+        background-color: rgba(0, 0, 250, 0.1);
+      }
+    `,
+  ],
   standalone: true,
-  imports: [],
+  imports: [CardComponent],
 })
 export class CityCardComponent implements OnInit {
-  constructor() {}
+  cities: City[] = [];
 
-  ngOnInit(): void {}
+  constructor(
+    private http: FakeHttpService,
+    private store: CityStore,
+  ) {}
+
+  ngOnInit(): void {
+    this.http.fetchCities$.subscribe((c) => this.store.addAll(c));
+
+    this.store.cities$.subscribe((c) => (this.cities = c));
```

Comment:
think declarative over imperative. 

prefer to always declare your properties on top of the class. And they will react to changes.

And try to use signals to get use to the new API